cmd/rcd/rcd.go

```diff
@@ -54,6 +55,20 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 			if err := checkRelease(rcflags.Opt.WebGUIUpdate); err != nil {
 				log.Fatalf("Error while fetching the latest release of rclone-webui-react %v", err)
 			}
+			if rcflags.Opt.NoAuth{
```

Comment:
This change looks like it completely disables --rc-no-auth so it needs to be conditional on something?  Or maybe removed?

Reply:
I think that serving the web GUI without auth is a security risk, so that is why it forces auth if ```--rc-no-auth``` is specified. Would we require ```--rc-no-auth``` for debugging?